sdk/metric/instrument.go

```diff
@@ -318,14 +318,14 @@ func newObservable[N int64 | float64](scope instrumentation.Scope, kind Instrume
 			unit:        u,
 			scope:       scope,
 		},
-		aggregators: agg,
+		inputs: in,
 	}
 }
 
 // observe records the val for the set of attrs.
 func (o *observable[N]) observe(val N, s attribute.Set) {
-	for _, agg := range o.aggregators {
-		agg.Aggregate(val, s)
+	for _, in := range o.inputs {
+		in.Async(val, s)
```

Comment:
```suggestion
	// Asynchronous measurements that do not have a context.
	ctx := context.Background() 
	for _, in := range o.inputs {
		in(ctx, val, s)
```

Reply:
> Asynchronous measurements do not have a context.

That is why the `Async` method exists, so no context is required here.